manim/mobject/numbers.py

```diff
@@ -6,14 +6,16 @@
 
 import numpy as np
 
+from manim.mobject.opengl_compatibility import ConvertToOpenGL
```

Comment:
Could you rewrite this as a relative import?

Reply:
Done.